tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
+		},
+		{
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.uk/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_uk.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.uk/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_in.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.in/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "REACHABLE_de.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://de.wikipedia.org/wiki/Wikipedia:Hauptseite",
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.co.uk/",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "UNREACHABLE_www.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
+			name:              "UNREACHABLE_in.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "http://www.google.co.in/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_uk.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "http://www.google.co.uk/",
+			shouldBeReachable: false,
```

Comment:
this is redundant.. the only change I made was to allow traffic for TLS ports without virtual service. HTTP was always allowed without a virtual service.

Reply:
are you sure?  I think i tested https lately without VS on 1.0.5 and it allows accessing to hosts I didn't specify in service entry.  Either way it is good to have tests created so we know what is expected to work :)

Reply:
ok you are right, i double checked, i was using https.   So the test here is also using https in service entry without VS and ensure it blocks the http google addr.